buildtools/multinet_envs.sh

```
@@ -1,31 +1,31 @@
-  export TARI_NETWORK=igor
-  export TARI_TARGET_NETWORK=testnet
-  export TARI_NETWORK_DIR=testnetdan
-  ;;
-*)
-  echo "mainnet"
-  export TARI_NETWORK=mainnet
-  export TARI_TARGET_NETWORK=mainnet
-  export TARI_NETWORK_DIR=mainnet
-  ;;
+case "${tagnet}" in
+  v*-pre.* | *esme* | *esmeralda* )
```

Shellcheck (0.10.0)

[warning] This pattern always overrides a later one on line 7.

(SC2221)